_data/simple-icons.json

```diff
@@ -443,7 +443,7 @@
         {
             "title": "Apache Cordova",
             "hex": "E8E8E8",
-            "source": "https://cordova.apache.org/artwork/"
+            "source": "https://apache.org/logos/"
```

Comment:
I'd leave the source as cordova.apache.com/artwork purely because there's a decent PSD of the monochrome version there.

Reply:
But the main purpose of #2887 is to unify all Apache icons under the same source.

Reply:
As mentioned in #2887 though:
> As with the Google review, I think that if a project maintains its own design guidelines with vector artwork (e.g., [Flink](https://flink.apache.org/material.html)) then they should take precedence.

The PSD available [here](https://cordova.apache.org/artwork/) does contain proper paths for the monochrome version, whereas the Apache link doesn't.

Reply:
I see... So let's return to the old source!